plugin/src/main/java/io/jenkins/plugins/analysis/warnings/PolyspaceParse.java

```diff
@@ -0,0 +1,42 @@
+package io.jenkins.plugins.analysis.warnings;
+
+import org.kohsuke.stapler.DataBoundConstructor;
+import org.jenkinsci.Symbol;
+import hudson.Extension;
+
+import io.jenkins.plugins.analysis.core.model.AnalysisModelParser;
+import io.jenkins.plugins.analysis.core.model.StaticAnalysisLabelProvider;
+import io.jenkins.plugins.analysis.core.model.SvgIconLabelProvider;
+
+/**
+ * Provides a parser and customized messages for Polyspace tool.
+ *
+ * @author Eva Habeeb
+ */
+public class PolyspaceParse extends AnalysisModelParser {
+    private static final long serialVersionUID = 5776036181982740586L;
+    private static final String ID = "PolyspaceParse";
+    private static final String ICON_NAME = "polyspace";
+
+    /** Creates a new instance of {@link PolyspaceParse}. */
+    @DataBoundConstructor
+    public PolyspaceParse() {
+        super();
+        // empty constructor required for stapler
+    }
+
+    /** Descriptor for this static analysis tool. */
+    @Symbol("PolyspaceParse")
```

Comment:
Seems that I missed that detail in my analysis-model PR.

Wouldn't that be a more appropriate symbol?

```suggestion
    @Symbol("polyspace")
```

Where does the `Parse` part came from? Is the tool called Polyspace Parse? Or is this suffix solely from your parser?

Either way: symbols need to use CamelCase, IDs need to use url-notation (all lower case). I can fix that right away in the analysis model as well. 

What do you think?

Reply:
The idea was to differentiate between polyspace commands and the parser command, just in case there is a command from polyspace tool defined in the same way as 'polyspace'.

Reply:
I see. Then let us use `polyspaceParser` if that is fine for you as well, since we already have several other tools that do it this way (PMD, AndroidLint).

Reply:
The appropriate ID would then be `polyspace-parser`.

Reply:
Yes polyspace-parser would be fine then. I will change that now.

Reply:
In analysis-model as well?

Reply:
Yes also in analysis-model. But you can change that ? Since the pull request is already approved.

Reply:
Yes, this library is under my control and only used by tools that I control. So no worries...

Reply:
I will also do the same for the Simulink check Parser in the other Pull request